prep-html-skeleton/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <title>HTML Skeleton Exercise</title>
+
+  <body>
+    <h1>I built a proper HTML Skeleton</h1>
+
+    <p>Hello, Html.</p>
+  </body>
+</html>
```

Comment:
It looks like you are missing a `<head>` tag here. The `<title>` should be inside of a `<head>` tag. The structure should look something like this

```
<!DOCTYPE html>
<html>
<head>
  <title>HTML Skeleton Exercise</title>
</head>
<body>
  <h1>I built a proper HTML skeleton!</h1>
</body>
</html>
```